docs/import/manage_disaster.js

```diff
@@ -864,26 +866,24 @@ function createAssetDropdown(
  * @param {Array<string>} propertyPath The path to the value of this element
  * @return {JQuery<HTMLElement>} The passed-in element, for chaining
  */
-function addChangeHandler(elt, propertyPath) {
+function addAssetDataChangeHandler(elt, propertyPath) {
   return elt.on(
       'change',
-      (event) =>
-          handleScoreAssetSelection($(event.target).val(), propertyPath));
+      (event) => handleAssetDataChange($(event.target).val(), propertyPath));
 }
 /**
  * Handles the user entering a value into score-related input
- * @param {string} val Value of input. 'None'/empty are treated like null (ugh)
+ * @param {string} val Value of input. empty strings are treated like null (ugh)
```

Comment:
Is this doc still right?

Reply:
i think so? if val evaluates to false we set it to null. Fixed a bit more, though, because we don't want false to turn into null.